### ISSUE TITLE
Accessibility Metadata

### ISSUE THREAD

**Issue by emily-phet** [phetsims/tasks#713]
In preparation for future publication of John Travoltage with accessibility feature:

The web standard for accessibility metadata can be found here:
https://www.w3.org/wiki/WebSchemas/Accessibility

With general example by Jon H. here: http://handbook.floeproject.org/AccessibilityMetadata.html

Suggestion from Jon H. for John Travoltage:
In the case of John Travoltage, the metadata may look something like this:

<metadata>
<meta property=“schema:accessibilityFeature”>longDescriptions</meta>
<meta property=“schema:accessibilityHazard”>noFlashing</meta>
<meta propery=“schema:accessibilityControl”>fullKeyboardControl</meta>
<meta propery=“schema:accessibilityControl”>fullMouseControl</meta>
<meta propery=“schema:accessibilityControl”>fullTouchControl</meta>
…
etc.
</metadata>

@ariel-phet - not sure who/where this goes, but we'll want to have this in place if possible.

**Comment by emily-phet** [comment]
Looking through the list of features, would structuralNavigation also be appropriate? Or not, would since the "primary" content is graphical with the navigated text being descriptions?

An observation: longDescription seems like the best fit from the list for the descriptions in the sim, but it’s a bit lackluster. Too bad there does not seem to be an existing feature in the list that really captures the dynamic quality of the sim descriptions. longDescription sounds like it was intended for some static sentences associated with a diagram (for example).

**Comment by jessegreenberg** [comment]
The context for this issue is that we would like to add search terms for accessible content in phet sims.  So perhaps this could be tracked in the website repository.

I am not sure how the website search works now or if it uses Metadata, but wherever the accesssible sim is located, the desired metadata could be added to that page?

**Comment by jessegreenberg** [comment]
10/27/16 dev meeting, we discussed that metadata would be added to sim pages.  I will move this issue to the website repo.  There we can determine things like what tags should be used to describe John Travoltage and other things.
